purchase_request/views/purchase_request_view.xml

```diff
@@ -0,0 +1,589 @@
+            <field name="model">purchase.request</field>
+            <field name="arch" type="xml">
+                <form string="Purchase Request" version="7.0">
+                <header>
+                    <button name="button_draft"
+                            states="to_approve,approved,rejected,done"
+                            string="Reset"
+                            type="object"
+                            groups="purchase_request.group_purchase_request_manager"/>
+                    <button name="button_to_approve" states="draft"
+                            string="Request approval" type="object"
+                            class="oe_highlight"/>
+                    <button name="button_approved"
+                            states="to_approve"
+                            string="Approve"
+                            type="object"
+                            class="oe_highlight"
+                            groups="purchase_request.group_purchase_request_manager"/>
+                    <button name="button_done"
+                            states="approved"
+                            string="Done"
+                            type="object"
+                            class="oe_highlight"
+                            groups="purchase_request.group_purchase_request_manager"/>
+                    <button name="button_rejected"
+                            states="to_approve,approved"
+                            string="Reject"
+                            type="object"
+                            groups="purchase_request.group_purchase_request_manager"/>
+                    <field name="state" widget="statusbar"
+                           statusbar_visible="draft,to_approve,approved,done,rejected"
+                           statusbar_colors='{"approved":"blue"}'/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button type="object"
+                            name="action_view_purchase_request_line"
+                            class="oe_stat_button"
+                            attrs="{'invisible':[('line_count', '=', 0)]}"
+                            icon="fa-list">
+                            <field name="line_count" widget="statinfo"
+                                   string="Lines"/>
+                        </button>
+                    </div>
+                    <div class="oe_edit_only">
+                        <label for="name" class="oe_inline"/>
+                        <label for="origin" class="oe_inline"/>
+                    </div>
+                    <h1>
+                        <field name="is_editable" invisible="1"/>
+                        <field name="name" class="oe_inline"
+                               attrs="{'readonly': [('is_editable','=', False)]}"/>
+                        <label string=","/>
+                        <field name="origin" class="oe_inline"
```

Comment:
I would move origin to a different place, it seem a little bit weird here for me.